docs/explanations/purging_services.md

```diff
@@ -0,0 +1,79 @@
+
+If a package is specific to the given service (`ubuntu-fips` for the FIPS related services, for example) then it will be removed during the purge process, when possible.
+
+The operation of reverting/uninstalling packages may leave the system in an undesired state, due to packages not meeting dependencies (partial or broken system caches) or to removal of kernels when alternative kernels are not working properly. Therefore, the feature is aimed at users/administrators who understand the impacts.
+
+
+## Using the CLI to disable and purge a service
+
+First of all it is important to note that this is an ***experimental feature***, so it is available but it is advisable to use at one's own discretion.
+
+To purge a service, use the CLI command:
+```bash
+$ sudo pro disable <service-name> --purge
+```
+
+Because of the risks involved, the expectation is that someone is looking at the screen while purging a service. To make it harder to use this feature on scripts, the `--assume-yes` flag is incompatible with the `--purge` flag.
+
+Because packages are being uninstalled/reinstalled, the execution of the `disable` command with `--purge` may take a while to complete.
+
+## What happens when I purge:
+
+### - livepatch, realtime-kernel, landscape
+
+Those services do not support the `--purge` operation as of now.
+
+### - anbox, cc-eal, cis/usg, esm-apps, esm-infra*, ros (and ros-updates)
+
+When those services are disabled with `--purge`, the sources and authentication will be removed first. Then, packages that are available only in the service repository will be removed from the system.
+
+The origin is detected based on the `origin` metadata defined on `apt` for all packages in this repository.
+
+Then, for the packages that are also present in the Ubuntu archives, there will be a downgrade to the highest possible version in the archive. Downgrade operations are the most common because the packages for specific services usually have higher version strings than the counterparts in the archive.
+
+Note that this does not mean that the installed versions after purging a service would be the same versions that were in the system before the given service was enabled. It is important to use the latest versions available in the archive to guarantee that dependency chains are resolved - in other words, that there are no broken dependencies between packages.
+
+```{note}
+\* Disabling esm-infra with --purge may involve removing a kernel, see below for more information
+```
+
+### - FIPS (and fips-updates/fips-preview)
+
+In the case of FIPS related services (and in some cases, esm-infra), there is an extra care when purging the packages: there may be Linux Kernel packages among the ones to be removed/downgraded.
+
+In this case, the Pro Client will look for at least one more kernel installed in the system. This check is performed looking at the `apt` packages installed, and matching the version strings to `vmlinu[z|x]` files in `/boot`.
```

Comment:
should kernel be capitalised or no? I've seen both and I'm not sure which way is right

Reply:
not capitalized - fixed